registry/donor/forms.py

```diff
@@ -37,3 +37,12 @@ def add_one_rodne_cislo(self, rodne_cislo):
 class NoteForm(FlaskForm):
     rodne_cislo = HiddenField(validators=[DataRequired()])
     note = TextAreaField("Poznámka k dárci")
+
+
+class IgnoreDonorForm(FlaskForm):
+    rodne_cislo = StringField("Rodné číslo", validators=[DataRequired()])
+    reason = TextAreaField("Důvod k ignoraci", validators=[DataRequired()])
```

Comment:
Důvod bych dal klidně jako nějaký delší StringField - uživatelé by se stejně neměli moc rozepisovat, protože pak by se to nevešlo do tabulky, a myslím si, že bude vypadat lépe, když budou všechny inputy na jednom řádku (textarea se rozlézá do výšky).

Reply:
Upraveno